src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -14,3 +15,61 @@ export function transform(formConfig, form) {
 export const supportingEvidenceOrientation = (
   <p>We’ll now ask you where we can find medical records or evidence about your worsened conditions after they were rated. You don’t need to turn in any medical records that you’ve already submitted with your original claim. <strong>We only need new medical records or other evidence about your condition after you got your disability rating.</strong></p>
 );
+
+// TODO: Change this into a function which returns the content with the appropriate [condition].
+export const evidenceTypesDescription = ({ formData }) => {
+  return (
+    <p>What supporting evidence do you have that shows how your {formData.disability.diagnosticText} <strong>has worsened since VA rated your disability</strong>?</p>
+  );
+};
+
+
+// Shows or collapses the "Which should I choose?" link at the bottom of the evidence types page
+// TODO: Investigate whether this should just use `expandUnder`
+export class evidenceTypeHelp extends React.Component {
```

Comment:
I think the name of this should be title cased. Components and class functions are normally title case.